projects/plugins/jetpack/tests/e2e/lib/flows/jetpack-connect.js

```diff
@@ -109,39 +103,42 @@ export async function syncJetpackPlanData( plan, mockPlanData = true ) {
 }
 
 export async function loginToWpSite( mockPlanData ) {
-	const siteUrl = getTunnelSiteUrl();
-	if ( ! siteUrl ) {
-		throw 'WOW, siteURL is empty!';
+	// Navigating to login url will always display the login form even if the user is already logged in
+	// To prevent unnecessary log in we navigate to Dashboard and check if logged in
+	await DashboardPage.visit( page, false );
+
+	if ( await WPLoginPage.isLoggedIn( page ) ) {
+		logger.step( 'Already logged in!' );
+		return;
```

Comment:
We probably want to set sandbox cookies in case we already logged in

Reply:
Good catch! I assumed it was already set, because we're saving the local storage, but we're saving it before setting the sandbox cookie.
Solved now.